### PR TITLE
Hook up create postings form

### PR DESCRIPTION
**Pull request by calvinsID**
Hook things up... Create post form works but some of the timestamps are hardcoded for now, we can fix it later. 
Also need to validate the form values on the backend and do error handling on the frontend

### CODE REVIEW

**Comment by annaher** [comment]
Looks really good! Need to fix the hardcoding for sure, kinda sucks that two out of the five fields that are actually shown on the card atm are hardcoded :/

File: server/routes/posts.js [hunk 1]
```diff
@@ -25,6 +26,53 @@ router.get('/posts/:id', (req, res) => {
...
+        seats: req.body.seats,
+        car: req.body.car,
+        comment: req.body.comment
+    }
+
+    return db.users.findOne({
+        where: {firebaseUniqueId: firebaseUniqueId}
+    }).then(user => {
+        if (!user || user.length === 0 || !user.dataValues) {
+            res.status(500).send('Invalid user: Did not create post');
```
Comment (Brendenvski) [inline server/routes/posts.js]: Mike and I decided that all the bodies should be of the form: {payload:, message:}, so in this case lets change it to {payload: {}, message: 'Invalid user: Did not create post'}

File: server/routes/posts.js [hunk 2]
```diff
@@ -25,6 +26,53 @@ router.get('/posts/:id', (req, res) => {
...
+                destination: req.body.destination,
+                origin: req.body.origin,
+                price: 20,
+                ride_time: 1569283200,
+                riders: [],
+                car_type: req.body.car,
+                available_seats: req.body.seats,
+                additional_info: req.body.comment
+            }).then(() => {
+                res.json({payload: 'Created post!'});
```
Comment (Brendenvski) [inline server/routes/posts.js]: maybe just return the created post here?

File: server/routes/posts.js [hunk 3]
```diff
@@ -25,6 +26,53 @@ router.get('/posts/:id', (req, res) => {
...
+            }).catch(error => {
+                console.error(error);
+                res.status(500).send('Error in /posts/create');
```
Comment (Brendenvski) [inline server/routes/posts.js]: same format as above

File: server/routes/posts.js [hunk 4]
```diff
@@ -25,6 +26,53 @@ router.get('/posts/:id', (req, res) => {
...
+            })
+        }
+    }).catch(error => {
+        console.error(error);
+        res.status(500).send('Error in /posts/create');
```
Comment (Brendenvski) [inline server/routes/posts.js]: same format s above

**Pull request merged**